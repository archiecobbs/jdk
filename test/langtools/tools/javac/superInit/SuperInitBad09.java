/*
 * @test /nodynamiccopyright/
 * @bug 8194743
 * @summary Test invalid reference to "this" prior to superclass construction
 *
 * @compile/fail/ref=SuperInitBad09.out -XDrawDiagnostics SuperInitBad09.java
 */
import java.util.concurrent.atomic.*;
class SuperInitBad09 extends AtomicReference<Object> {

    SuperInitBad09() {
        super(AtomicReference.this);
    }
}
