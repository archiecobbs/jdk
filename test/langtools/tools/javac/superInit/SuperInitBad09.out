SuperInitBad09.java:12:30: compiler.err.not.encl.class: java.util.concurrent.atomic.AtomicReference
1 error
